src/ebpf_vm_isa.hpp

```diff
@@ -87,6 +87,10 @@ enum {
     INST_ALU_OP_ARSH = 0xc0,
     INST_ALU_OP_END = 0xd0,
     INST_ALU_OP_MASK = 0xf0,
+
+    INST_LD_MODE_IMM = 0x00,      // 64-bit immediate value
```

Comment:
```suggestion
    INST_LD_MODE_IMM = 0x0,       // 64-bit immediate value
```
For consistency with other lines below